smart_contract/scripts/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './constants'
+export * from './setup'
```

Comment:
why the exports? I don't see them being used anywhere?

Reply:
They are imported by Faucet.test.ts to deploy the contract

Reply:
I see, missed that!